src/style.css

```diff
@@ -1,3 +1,69 @@
 body {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: Roboto, Arial, sans-serif;
```

Comment:
CSS: It's recommended to use fallback fonts. This can be done by specifying a list of fonts in the `font-family` property. If the first font is not available, the browser will try to use the next one in the list.